myCafySvc/dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:current-alpine
+LABEL maintainer="yourMail@yourHost.com"
+
+WORKDIR /usr/app
+
+RUN apk add --no-cache python3 py3-pip alpine-sdk git
+#RUN npm install git 
+RUN npm install noble
+RUN npm install prettier
+RUN npm install -g node-gyp
+#RUN npm install bluetooth-hci-socket
+RUN npm install @abandonware/bluetooth-hci-socket
```

Comment:
```suggestion
```

I don't think we need to install all this packages. They are already listed in the package.json. Or is there other reasons?